src/services/users.js

```diff
@@ -77,8 +79,38 @@ const signInUserByEmail = async (email, password) => {
 	return { token, userId: user.id }
 }
 
+/**
+ * service to invite user
+ * @param {string} email - email of the user to be invited
+ * @returns {object}
+ */
+const inviteUser = async (email) => {
+
+	const token = jwt.sign({ email }, 
+		config("TOKEN_SECRET"),
+		 {expiresIn: "7d"})
+
+	const url = `${frontendUrl}/signup?token=${token}`
+
+	const text = `Hi,
+	\n\nPlease click on the following link to complete your registration:
+	\n${url}\n\nIf you did not request this, please ignore this email.\n`
+
+	await emailLib({ to: email, subject: "Invitation to signup", text })
+
+	const { rows } = await db.query(
+		`INSERT INTO user_invites ("email") VALUES ($1) RETURNING *`, [email])
```

Comment:
@itzgeebee thinking about this, let's insert before sending the email. if for some reason we send the email and the insert doesn't work the invite link will not work anyway